src/utils/ProteomicsLFQ.cpp

```diff
@@ -514,23 +524,33 @@ class ProteomicsLFQ :
   {
     if (feature_maps.size() > 1) // do we have several maps to align / link?
     {
-      Param ma_param = getParam_().copy("Alignment:", true);
-      writeDebug_("Parameters passed to MapAlignmentAlgorithmIdentification algorithm", ma_param, 3);
-
-      // Determine reference from data, otherwise a change in order of input files
-      // leads to slightly different results
-      const int reference_index(-1); // set no reference (determine from data)
-      MapAlignmentAlgorithmIdentification aligner;
-      aligner.setLogType(log_type_);
-      aligner.setParameters(ma_param);
+      Param mat_param = getParam_().copy("Alignment:", true);
```

Comment:
is it correctly copied back here with the new subsection?

Reply:
no that was one issue! thanks